src/screens/monitor/components/documentTable/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { GenericTable } from "components/genericTable"
+import { GenericTableRow } from 'components/genericTableRow';
+import { DocumentDto } from 'services/api/dtos';
+
+interface DocumentsTableProps {
+  documents: DocumentDto[];
+};
+
+export const DocumentsTable = (props: DocumentsTableProps) => {
+  const { documents } = props;
+
+  const columns = [
```

Comment:
This can be moved outside of the component to avoid array reinitialisation on subsequent rerenders